packages/fiori-freestyle-writer/templates/basic/extend/webapp/manifest.json

```
@@ -26,7 +26,7 @@
             "routes": [
                 {
                     "name": "Route<%- template.settings.viewName %>",
-                    "pattern": "Route<%- template.settings.viewName %>",
+                    "pattern": ":?query:",
```

```suggestion
                    "pattern": "/:?query:",
```
I was a bit confused when I read this first. Since I expect the idea is to set the default route to the app root would this adding the '/' be more obvious?

---

Hmm, not sure. I wanted to be consistent with what we create in Fiori elements, so I copied it from there.

---

OK. Not a big deal.